### DISCUSSION TITLE
do we have any variable to increase 'Block name can be a maximum of 60  characters' instead of 32  ?

### DISCUSSION THREAD

**Discussion by nagachakithmck** [Azure/ipam#143] [Q&A] [answered]
Do we have any variable to acheive this ? or cloning the github code and editing the source code is only option ?

**Comment by DCMattyG** [comment]
Good morning @nagachakithmck. At this time, there isn't a variable to adjust max length of a Block name. In choosing the current limits we tried to create a balance where Block names could be descriptive enough, but not start bleeding out of their UI elements in the browser. There is also the Description field so you can provide greater detail on the purpose of the Block as opposed to trying to fit all of that context in the Block name alone.

That said, I'm **always** open to feedback if the currently enforced length isn't enough. Are you able to provide some examples of how you are naming Blocks that is causing them to exceed 32 characters?

**Reply by nagachakithmck** [reply]
thanks for reply @DCMattyG. we are trying to migrate from our existing system to ipam. so the names of CIDR block like below

demoteam2-acc-az-12-westeurope-cidr
demoteam9-acc-az-12-westeurope-cidr

**Reply by DCMattyG** [reply, marked as answer]
Hi @nagachakithmck, this has been fixed with PR #153. You should be good to go my friend! Let me know if you have any troubles at all.

**Reply by nagachakithmck** [reply]
many thanks @DCMattyG
